README.md

```diff
@@ -2,6 +2,7 @@
 
 Cartopy
 -------
+[![Build Status](https://api.travis-ci.org/repositories/SciTools/cartopy.svg?branch=master)](http://travis-ci.org/SciTools/cartopy/branches)
```

Comment:
@marqh I thought this would be better underneath the title, rather than above it.  Seems a bit more consistent with Iris this way.